.circleci/config.yml

```diff
@@ -59,6 +59,25 @@ jobs:
       - run:
           name: Running server tests
           command: cd server && yarn test
+  main:
```

Comment:
the job `main` may not be as descriptive as what it does

Reply:
Suggestion?

Reply:
`test_serve`?